integration/spark2/src/main/scala/org/apache/carbondata/indexserver/DistributedShowCacheRDD.scala

```diff
@@ -43,15 +44,25 @@ class DistributedShowCacheRDD(@transient private val ss: SparkSession, tableName
 
   override def internalCompute(split: Partition, context: TaskContext): Iterator[String] = {
     val dataMaps = DataMapStoreManager.getInstance().getAllDataMaps.asScala
+    val tableList = tableName.split(",")
     val iterator = dataMaps.collect {
       case (table, tableDataMaps) if table.isEmpty ||
-                                     (tableName.nonEmpty && tableName.equalsIgnoreCase(table)) =>
+                                     (tableName.nonEmpty && tableList.contains(table)) =>
         val sizeAndIndexLengths = tableDataMaps.asScala
-          .map(_.getBlockletDetailsFetcher.getCacheSize)
-        // return tableName_indexFileLength_indexCachesize for each executor.
-        sizeAndIndexLengths.map {
-          x => s"$table:$x"
-        }
+          .map { dataMap =>
+            if (!dataMap.getDataMapSchema.getProviderName
+              .equals(DataMapClassProvider.BLOOMFILTER.getShortName)) {
```

Comment:
I don't get why we are specifically checking for `BLOOMFILTER`. We are not supposed to check anything specific

Reply:
changed the condition to check if dataMapName is null (for BlockletDataMap)